lib/create-http-client.js

```diff
@@ -27,6 +29,10 @@ export default function createHttpClient (axios, options) {
   const defaultConfig = {
     insecure: false,
     retryOnError: true,
+    logEmitter: new EventEmitter(),
```

Comment:
I think a better option is to merge `logEmitter` and `errorEmitter` to one `logEmitter` that emmits `error`, `warning` and `info`